.github/workflows/release.yaml

```diff
@@ -0,0 +1,122 @@
+name: Gingersnap Operator - Build and publish
+
+on:
+  workflow_dispatch:
+    inputs:
+      ref:
+        description: "The Operator branch/tag to checkout when creating the image."
+        required: true
+        default: "release_operator_image"
+      
+
+
+env:
+  GITHUB_TOKEN: ${{ secrets.PAT_TOKEN }}   
+  release-version: ${{ github.event.inputs.ref }}
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Configure Git
+        run: |
+          git config --global user.name "infinispan-qe-bot"
+          git config --global user.email "q*@infinispan.org"
+
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          ref: ${{ github.event.inputs.ref }}
+          path: operator
+
+      - name: Tag the release
+        run: |
+          git tag -a v${{ env.release-version}} -m "Release Operator:: ${{ env.release-version}}"
+          git push origin --tags
+        
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v2
+
+      - name: Set up Docker Buildx
+        id: buildx
+        uses: docker/setup-buildx-action@v2
+
+      - name: Login to Quay
+        uses: docker/login-action@v2
+        with:
+          registry: quay.io
+          username: ${{ secrets.QUAY_USERNAME }}
+          password: ${{ secrets.QUAY_TOKEN }}
+
+      - name: Build and push
+        uses: docker/build-push-action@v3
+        with:
+          context: .
+          platforms: linux/amd64,linux/arm64
+          push: true
+          tags: quay.io/gingersnap/operator:${{ env.release-version }}
+          build-args: |
+            OPERATOR_VERSION=${{ env.release-version }}
+
+      - name: Create OLM Bundle
+        run:  |
+            make operator-sdk
+            make bundle VERSION=${{ env.release-version }} CHANNELS=alpha DEFAULT_CHANNEL=alpha IMG=quay.io/gingersnap/operator:${{ env.release-version }}
+
+      - name: Checkout community-operators repo
+        uses: actions/checkout@v2
+        with:
+          repository: k8s-operatorhub/community-operators
+          path: community-operators
+          ref: main
+
+      - name: Copy contents of bundle/ 
+        run: |
+          mkdir -p community-operators/operators/gingersnap/${{ env.release-version }}
+          cp -R bundle/* community-operators/operators/gingersnap/${{ env.release-version }}
+          git add community-operators/operators/gingersnap/${{ env.release-version }}
+          git commit -m "Operator Bundle ${{ env.release-version }}"
```

Comment:
This commit will exist in the community-operators repo that is full of other operator bundles, therefore we should give this commit a more meaningful name.

How about: `"Gingersnap ${{ env.release-version }}"`?